test/units/module_utils/basic/test__log_invocation.py

```diff
@@ -22,75 +22,62 @@
 
 import sys
 import json
-from io import BytesIO, StringIO
 
-from ansible.compat.six import PY3
-from ansible.utils.unicode import to_bytes
+from units.mock.procenv import swap_stdin_and_argv
```

Comment:
I actually tried that first, but it gripes about relative imports in non-packages when I run the tests- damned if you do, damned if you don't...